.github/workflows/gh-packages.yml

```diff
@@ -2,7 +2,8 @@ name: Publish to GitHub Packages
 on:
   push:
     branches:
-      - develop
+      - EGRC-492
```

Comment:
flip this back to `develop` now that we're ready to merge and i'm set to approve.